src/components/pages/donation_form/DonationReceipt/usePersonalDataSectionEventHandlers.ts

```diff
@@ -81,6 +82,13 @@ export function usePersonalDataSectionEventHandlers(
 		}
 	} );
 
+	watch( () => store.state.payment.values.type, ( newType: string ) => {
```

Comment:
It's a bit annoying that we need to add extra watchers for this, but it'll get a bit better when we have the new IBAN field sorted.